packages/web-app/src/utils/config.ts

```diff
@@ -61,25 +73,29 @@ export const PLUGIN_IDS: { [K in SupportedNetwork]: PluginsDeployment } = {
         creditDelegation: "my-cool-plugin-6016",
         subgovernance: "my-cool-plugin-6002",
         uniswapV3: "my-cool-plugin-6017",
-        vault: "my-cool-plugin-6005"
+        vault: "my-cool-plugin-6005",
+        pwn: "",
```

Comment:
Address is missing

Reply:
But this empty string, at line 77, is on the IDs array. Not in the addresses array. Are you suggesting that we should also include that address as an ID?

Reply:
Yes, line 77 should be the name of the plugin  registered in the plugin preprocessor, for pwn should be my-cool-plugin-6020